src/wp-includes/blocks/search.php

```diff
@@ -16,7 +16,7 @@
  *
  * @return string The search block markup.
  */
-function render_block_core_search( $attributes, $content, $block ) {
+function render_block_core_search( $attributes ) {
```

Comment:
The removed parameters are still documented. The documentation should be removed.

Side-note: Isn't removing these without handling/renaming to `$deprecated` a BC break? If someone previously called this function with three arguments, it won't have the same behaviour as before, right?

Reply:
@c4rl0sbr4v0  Can you please follow-up with a PR in Gutenberg to address the comments that are related to the changes here

Reply:
> Side-note: Isn't removing these without handling/renaming to $deprecated a BC break? If someone previously called this function with three arguments, it won't have the same behaviour as before, right?


As it is a `render_callback`, those functions should have always those three params available, regardless of we use them or not in core blocks. I tried this code with them removed and worked as expected.

```php
function plugin_new_search($attributes, $content, $block) {
  var_dump($block); // print $block object.
}

add_filter('register_block_type_args', function ($settings, $name) {
    if ($name == 'core/search') {
        $settings['render_callback'] = 'plugin_new_search';
    }
    return $settings;
}, null, 2);
```

>@c4rl0sbr4v0 Can you please follow-up with a PR in Gutenberg to address the comments that are related to the changes here

Done in https://github.com/WordPress/gutenberg/pull/58391